app/views/epitaphs/share.html.erb

```diff
@@ -9,3 +9,67 @@
   <div><%= link_to t(:write_yours_from_share), root_path %>
 <% end %>
 
+<div id="threedeeplayland">
+    <script>
+    var SOONRIP = SOONRIP || {}
+    SOONRIP['ep'] = { name: "<%= @epitaph.name %>",
+                    epitaph:"<%= @epitaph.text %>"};
+    </script>
+
+    <div oncontextmenu="return false;" id="viewerContext" style="width: 50 vw; height: 50vh"></div>
```

Comment:
Chrome complains that `50 vw` is invalid. Maybe should be `50vw`? Honestly I have never seen vw as a unit in CSS before and I have no idea what this is. Googling now...